src/NLog.Mongo/MongoTarget.cs

```diff
@@ -161,6 +163,26 @@ public string CollectionName
         /// </summary>
         public bool IncludeEventProperties { get; set; }
 
+        /// <summary>
+        /// Gets or sets if TLS should be used when connecting to MongoDB
+        /// </summary>
+        public bool UseTls { get; set; }
+
+        /// <summary>
+        /// Gets or sets the client certificate to use when connecting to MongoDB
+        /// </summary>
+        public string ClientCertificate { get; set; }
```

Comment:
Recommended to use NLog `Layout` instead of  `string`. If `ClientCertificate` is a file-path, then maybe include that in the property-name.